### PR TITLE
Changes for NewsML-G2 version 2.35

### PR DESCRIPTION
**Pull request by bquinn**
* Updating dev schema to 2.35. Fixes iptc/news-architecture-dev#157

### CODE REVIEW

File: specification/NewsML-G2_2.35-spec-All-Power.xsd
Comment (davecompton) [inline specification/NewsML-G2_2.35-spec-All-Power.xsd:1]: @bquinn 
NewsML-G2 2.35: dataMining added ... that was in 2.34
Same change/comment in other files ...

File: examples/NewsML-G2_2.35-spec-All-Power.xsd
Comment (davecompton) [inline examples/NewsML-G2_2.35-spec-All-Power.xsd:1]: @bquinn 
NewsML-G2 2.35: dataMining added ... was 2.34

File: specification/XML-Schema_FileVersion_1/NewsML-G2_2.35-spec-All-Power_1.xsd
Comment (davecompton) [inline specification/XML-Schema_FileVersion_1/NewsML-G2_2.35-spec-All-Power_1.xsd:1]: @bquinn 
NewsML-G2 2.35: dataMining added ... was 2.34

File: specification/XML-Schema_FileVersion_1/NewsML-G2_2.35-spec-Framework-Power_1.xsd
Comment (davecompton) [inline specification/XML-Schema_FileVersion_1/NewsML-G2_2.35-spec-Framework-Power_1.xsd:1]: @bquinn 
NewsML-G2 2.35: dataMining added ... was 2.34

File: specification/individual/NewsML-G2_2.35-spec-Framework-Power.xsd
Comment (davecompton) [inline specification/individual/NewsML-G2_2.35-spec-Framework-Power.xsd:1]: @bquinn 
NewsML-G2 2.35: dataMining added ... was 2.34

File: tests/schema_versions/NewsML-G2_2.35-spec-All-Power.xsd
Comment (davecompton) [inline tests/schema_versions/NewsML-G2_2.35-spec-All-Power.xsd:1]: @bquinn 
NewsML-G2 2.35: dataMining added ... was 2.34

**Review by davecompton — APPROVED** [review]
Looks good

File: dev-schema/NewsML-G2dev_0.10_nar235.xsd [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+	<xs:annotation>
+		<xs:documentation>This XML Schema is created for a NewsML-G2 development namespace with the URI https://iptc.org/std-dev/nar/. It imports a NewsML-G2 2.x XML Schema and its elements build on attributes, attribute groups and elements defined by it.
+		</xs:documentation>
+		<xs:documentation>NewsML-G2 Dev 0.10 XML Schema - UNDER CONSTRUCTION</xs:documentation>
+		<xs:documentation>Date of creation of this XML Schema document revision: 2024-08-23</xs:documentation>
+		<xs:documentation>Last change: update to be based on 2.35 schema</xs:documentation>
+		<xs:documentation>Created by Michael Steidl</xs:documentation>
+		<xs:documentation>Revised by Brendan Quinn</xs:documentation>
+	</xs:annotation>
+	<xs:import namespace="http://iptc.org/std/nar/2006-10-01/" schemaLocation="NewsML-G2_2.34-spec-All-Power.xsd"/>
```
Comment (bquinn) [inline dev-schema/NewsML-G2dev_0.10_nar235.xsd]: should be 2.35

File: tests/schema_versions/NewsML-G2dev_0.10_nar235.xsd [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+	<xs:annotation>
+		<xs:documentation>This XML Schema is created for a NewsML-G2 development namespace with the URI https://iptc.org/std-dev/nar/. It imports a NewsML-G2 2.x XML Schema and its elements build on attributes, attribute groups and elements defined by it.
+		</xs:documentation>
+		<xs:documentation>NewsML-G2 Dev 0.10 XML Schema - UNDER CONSTRUCTION</xs:documentation>
+		<xs:documentation>Date of creation of this XML Schema document revision: 2024-08-23</xs:documentation>
+		<xs:documentation>Last change: update to be based on 2.35 schema</xs:documentation>
+		<xs:documentation>Created by Michael Steidl</xs:documentation>
+		<xs:documentation>Revised by Brendan Quinn</xs:documentation>
+	</xs:annotation>
+	<xs:import namespace="http://iptc.org/std/nar/2006-10-01/" schemaLocation="NewsML-G2_2.34-spec-All-Power.xsd"/>
```
Comment (bquinn) [inline tests/schema_versions/NewsML-G2dev_0.10_nar235.xsd]: should be 2.35

**Comment by bquinn** [comment]
I tried loading the schema into XMLSpy on Friday, and discovered that there were many other places that also needed to have complexContent changed to simpleContent to avoid XML Spy warnings and schema re-writes. So I have now applied those changes, and XMLSpy now opens the schema with no warnings. (It still re-writes the schema but all changes are to whitespace and formatting.)

I have updated the Change Request accordingly: http://dev.iptc.org/G2-CR00220-change-generator-to-simpleContent

Note that a similar change was made made in NAR 1.2, according to comments eg on timestamp: `<!-- NAR 1.2 erratum 1 correction: on timestamp, complexContent replaced by simpleContent -->`

That raises the question, should this be an erratum release or a full release? We're not changing any functionality...

**Comment by davecompton** [comment]
Hmm ... as it only contains non-functional / whitespace changes, it sounds like v2.34_2.
I guess that's a lot to rework at this point ... ?

**Pull request closed without merging**